### PR TITLE
CapsuleContent [CompEval] Destructive tests SCA fixes

### PR DESCRIPTION
**Pull request by damoore044**
### Purpose (component eval)
Update to rhel9 repos/content
Resolve intermittent failures for destructive tests (Stream and 6.15.z):
- test_positive_sync_without_deadlock
- test_positive_sync_without_deadlock_after_rpm_trim_changelog

There are in-progress tasks for some components, lots of actions are performed too early inside an SCA org
```
trigger: test-robottelo
pytest: tests/foreman/destructive/test_capsulecontent.py
```

### CODE REVIEW

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_capsulecontent.py

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7744
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/destructive/test_capsulecontent.py --external-logging
Test Result : ======= 1 failed, 1 passed, 495 warnings, 1 error in 4877.90s (1:21:17) ========
```

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_capsulecontent.py

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7824
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/destructive/test_capsulecontent.py --external-logging
Test Result : ======= 1 failed, 1 passed, 486 warnings, 1 error in 4751.88s (1:19:11) ========
```

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_capsulecontent.py

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7893
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/destructive/test_capsulecontent.py --external-logging
Test Result : ============ 1 failed, 1 passed, 547 warnings in 4958.99s (1:22:38) ============
```

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_capsulecontent.py

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7896
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/destructive/test_capsulecontent.py --external-logging
Test Result : ================ 2 passed, 1216 warnings in 9519.93s (2:38:39) =================
```

**Comment by damoore044** [comment]
trigger: test-robottelo
pytest: tests/foreman/destructive/test_capsulecontent.py

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7898
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/destructive/test_capsulecontent.py --external-logging
Test Result : ================ 2 passed, 1274 warnings in 10006.13s (2:46:46) ================
```

**Review by pondrejk — APPROVED** [review]

**Review by sambible — APPROVED** [review]

File: tests/foreman/destructive/test_capsulecontent.py [hunk 1]
```diff
@@ -36,61 +41,112 @@ def test_positive_sync_without_deadlock(
...
         basearch=constants.DEFAULT_ARCHITECTURE,
         org_id=function_sca_manifest_org.id,
-        product=constants.REPOS['rhscl7']['product'],
-        repo=constants.REPOS['rhscl7']['name'],
-        reposet=constants.REPOSET['rhscl7'],
-        releasever=constants.REPOS['rhscl7']['releasever'],
+        product=constants.REPOS['rhel9_bos']['product'],
+        repo=constants.REPOS['rhel9_bos']['name'],
+        reposet=constants.REPOSET['rhel9_bos'],
+        releasever=constants.REPOS['rhel9_bos']['releasever'],
```
Comment (damoore044) [inline tests/foreman/destructive/test_capsulecontent.py:65]: @vsedmik recommended using a repo with more content counts than `rhel9_bos`.
Agreed, perhaps a repository like `rhel9_aps` or multiple repos to achieve 10,000+ content count.

**Pull request merged**